apps/src/templates/projects/generateFakeProjects.js

```diff
@@ -16,32 +16,30 @@ export function generateFakePersonalProjects(n) {
 
 export const stubFakePersonalProjectData = [
   {
-    projectName: "Personal Project 1",
+    name: "Personal Project 1",
```

Comment:
just curious -- why did projectName and isPublished change to name and publishedAt?

Reply:
I set up the Storybook data first, then fetched the real data in #23755 where I changed the field names.  This just makes them consistent.